web_external/js/views/widgets/WMSFeatureInfoWidget.js

```diff
@@ -0,0 +1,110 @@
+minerva.views.WmsFeatureInfoWidget = minerva.View.extend({
+
+    callInfo: function (layer_idx, coords) {
+        if (this.layers.length > 0) {
+            var url = this.getUrl(layer_idx, coords);
+            var layer_name = this.layers[layer_idx].layerName;
+            var panel = this;
+            $.ajax({
+                url: url,
+                jsonp: false,
+                async: false,
```

Comment:
can we make this `async: true`?  Local testing shows me it works, but maybe there are other considerations?